.circleci/config.yml

```diff
@@ -337,7 +374,13 @@ commands:
       - run:
           name: Run Navigation Core SDK Unit Tests and generate Jacoco test report
           command: |
-            make core-unit-tests-jacoco
+            if [[ "$CIRCLE_BRANCH" == "main" ]]; then
```

Comment:
OK, but do we actually trigger anything on main (**after** merge)?

Reply:
Yes